src/test/java/org/openmrs/contrib/qaframework/automation/Steps.java

```diff
@@ -25,6 +26,7 @@
 	protected ClinicianFacingPatientDashboardPage dashboardPage;
 	protected By patientHeaderId = By.cssSelector("div.identifiers span");
 	protected InitialSetupPage initialSetupPage;
+	protected PatientVisitsDashboardPage visitsDashboardPage;
```

Comment:
Added` PatientVisitDashboardPage` to be global for allowing child node classes to extend its functionality than declaring it in each child class that may need it.